pkg/operator/ceph/cluster/nodedaemon/reconcile.go

```diff
@@ -227,20 +227,22 @@ func (r *ReconcileNode) createOrUpdateNodeDaemons(node corev1.Node, tolerations
 				return errors.Wrapf(err, "ceph exporter reconcile failed on op %q", op)
 			}
 		} else {
-			logger.Debugf("ceph exporter successfully reconciled for node %q. operation: %q", node.Name, op)
-			// create the metrics service
-			service, err := MakeCephExporterMetricsService(cephCluster, exporterServiceMetricName, r.scheme)
-			if err != nil {
-				return err
-			}
-			if _, err := k8sutil.CreateOrUpdateService(r.opManagerContext, r.context.Clientset, cephCluster.Namespace, service); err != nil {
-				return errors.Wrap(err, "failed to create ceph-exporter metrics service")
-			}
+			if cephVersion.IsAtLeast(cephver.CephVersion{Major: 18, Minor: 0, Extra: 0}) {
```

Comment:
Let's add a short comment on "why" Ceph version 18 is required here

Reply:
Since this version is used in two places, let's create a package var for it, and then add the comment there about the reason.

Reply:
Added a `MinVersionForCephExporter` var